components/fxa-client/src/lib.rs

```diff
@@ -60,9 +60,9 @@ pub use push::{
 pub use token::{AccessTokenInfo, AuthorizationParameters, ScopedKey};
 
 /// Result returned by internal functions
-type Result<T> = std::result::Result<T, Error>;
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
Does the internal error need to be `pub`? Looks like all calls use the public API right?